### ISSUE TITLE
Decreased beta-mannosidase

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7584]
**Preferred term label:**
Decreased beta-mannosidase

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 248510 

PMID: 18565776  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
